ads/_config.js

```diff
@@ -252,6 +252,11 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'adyoulike': {
+    consentHandlingOverride: true,
+    prefetch: 'https://pixels.omnitagjs.com/amp.js',
```

Comment:
could you pls follow [this guideline](https://github.com/ampproject/amphtml/blob/master/ads/README.md#other-tips) to implement render-start API?

Reply:
done :)